app/components/documents/document-card.hbs

```diff
@@ -127,6 +127,7 @@
               (and
                 @isEditable
                 (user-may "manage-documents")
+                this.mayEditOrDelete
               )
```

Comment:
- This hides the entire dropdown button so no need to hide the individual options later on.
- `hasSignFlow` or similar here.

- My opinion (to discuss) is to maybe have an option with info as to why there are no actions available (either an option with info, or replace the dropdown icon with an info icon. Perhaps to discuss with the users.